### PR TITLE
Update arubaos-cx.yaml to remove historical temps as limits

### PR DESCRIPTION
**Pull request by TridTech**
Removing low_limit and high_limit that was setting the values to the historical low and high temperature recorded by the switch.

Example from the MIB (one of two values removed from the discovery file):
arubaWiredTempSensorMinTemp OBJECT-TYPE
      SYNTAX      Integer32
      UNITS       "millidegrees Celsius"
      MAX-ACCESS  read-only
      STATUS      current
      DESCRIPTION "Historic minimum temperature of the temperature sensor."

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [ ] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [ ] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.
- [ ] If my Pull Request makes discovery/polling/yaml changes, I have added/updated [test data](https://docs.librenms.org/Developing/os/Test-Units/).

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: includes/definitions/discovery/arubaos-cx.yaml [hunk 1]
```diff
@@ -28,8 +28,6 @@ modules:
                     descr: arubaWiredTempSensorName
                     divisor: 1000
                     value: arubaWiredTempSensorTemperature
-                    low_limit: arubaWiredTempSensorMinTemp
-                    high_limit: arubaWiredTempSensorMaxTemp
```
Comment (PipoCanaja) [inline includes/definitions/discovery/arubaos-cx.yaml:32]: Right, historical limits should not be used.

**Comment by PipoCanaja** [comment]
Hi @TridTech 
Could you re-run the tests please ? (`save-test-data.php -o arubaos-cx`) ?
Thx

**Comment by TridTech** [comment]
> Hi @TridTech Could you re-run the tests please ? (`save-test-data.php -o arubaos-cx`) ? Thx

The original tests were not from me, and I don't have access to that model of switch.  I may be able to submit new tests for a different model, but I have not been successful in getting snmpsim running in the past.

**Comment by mpikzink** [comment]
I found this in the manual from a HPE Aruba Networking CX 6400

> Operating temperature 32°F to 113°F (0°C to 45°C), up to 5,000 feet 32°F to 104°F (0°C to 40°C), 5,001 to 10,000 feet 1°C de‑rating per 1,000 feet above 5,000 feet.
https://www.hpe.com/psnow/doc/a00085158enw

What do you think of hard code the limit values, since they are not included in the MIBs? I have not yet found any other limit values in the CX portfolio.

**Comment by mpikzink** [comment]
you're right, I hadn't thought about the ASICs and so on.

**Comment by murrant** [comment]
The tests reflect the bad data, so you will need to update the test data. I think this should do it `./scripts/save-test-data.php -o arubaos-cs -m sensors`

**Comment by laf** [comment]
I've re-generated the test data.

**Pull request merged**